component.html

```diff
@@ -305,7 +305,156 @@ <h3 class="avatar-heading">Floating action Button</h3>
 </iframe>
     </section>
 
-    
+    <!-- --- Card ---  -->
+
+    <section class="main-title" id="cards">
+      <h2 class="avatar-heading">Cards</h2>
+      <p class="component-navbar-item">
+        Card are used to show user related data collectively, like product details.
+      </p>
+      <h3 class="avatar-heading">Cards with badges</h3>
```

Comment:
change class name "avatar-heading"  to a generic name.

Reply:
done